content/en/docs/tasks/configure-pod-container/security-context.md

```diff
@@ -449,8 +449,7 @@ SELinux label of a volume instantly by using a mount option
 
 To benefit from this speedup, all these conditions must be met:
 
-* Alpha feature gates `ReadWriteOncePod` and `SELinuxMountReadWriteOncePod` must
-  be enabled.
+* Alpha feature gate `SELinuxMountReadWriteOncePod` must be enabled.
```

Comment:
I think under https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes we should also add the `feature-state` tag.

Reply:
Removed this feature here since it's in beta and added the feature tag to where `ReadWriteOncePod` is outlined in the PV docs.